File: DS-Algo/inflearn_algo_basic/7/7.cpp
#include<iostream>
#include<string>
#include<cctype>
using namespace std;

int main()
{
    string infected_word;
    getline(cin,infected_word);

    string collected_word;
    for(char c : infected_word)
    {
        if(!isspace(c))
            collected_word += tolower(c);
    }
    cout << collected_word;

    return 0;
}

File: DS-Algo/inflearn_algo_basic/7/7.py
# infected_word = input()

# collected_word = ''
# for c in infected_word:
#     if not c.isspace():
#         collected_word +=c.lower()
# print(collected_word)

infectd_word = input()

#collected_word = "".join(c.lower() for c in infectd_word if not c.isspace())
collected_word = infectd_word.replace(' ','').lower()

print(collected_word)